vumi/transports/smpp/tests/test_protocol.py

```diff
@@ -405,6 +405,26 @@ def test_submit_sm_multipart_udh(self):
         stored_ids = yield self.lookup_message_ids(protocol, seq_numbers)
         self.assertEqual(['abc123'] * len(seq_numbers), stored_ids)
 
+    @inlineCallbacks
+    def test_submit_sm_multipart_udh_short(self):
+        # if message doesn't need splitting, then don't use the UDH
+        transport, protocol = yield self.setup_bind(config={
+            'send_multipart_udh': True,
+        })
+        short_message = 'This is a short message.'
+        seq_nums = yield protocol.submit_sm(
+            'abc123', 'dest_addr', short_message=short_message)
+        [submit_sm] = yield wait_for_pdus(transport, 1)
```

Comment:
Cool! Didn't know you could do this in Python.

Reply:
Heh, also a feature of my copy/paste from line 322. (which also copied the wrong protocol call... will fix.)